src/core/server/saved_objects/migrations/integration_tests/incompatible_cluster_routing_allocation.test.ts

```diff
@@ -9,13 +9,19 @@
 import Path from 'path';
 import fs from 'fs/promises';
 import JSON5 from 'json5';
+import { REPO_ROOT } from '@kbn/utils';
+import { Env } from '@kbn/config';
+import { getDocLinksMeta } from '@kbn/doc-links';
+import { getEnvOptions } from '../../../config/mocks';
 import * as kbnTestServer from '../../../../test_helpers/kbn_server';
 import { Root } from '../../../root';
 import { ElasticsearchClient } from '../../../elasticsearch';
 import { LogRecord } from '@kbn/logging';
 import { retryAsync } from '../test_helpers/retry_async';
 
 const logFilePath = Path.join(__dirname, 'incompatible_cluster_routing_allocation.log');
+const env = Env.createDefault(REPO_ROOT, getEnvOptions());
+const docVersion = getDocLinksMeta({ kibanaBranch: env.packageInfo.branch }).version;
```

Comment:
What kind of version does this field return? I'm asking cause I've checked the resulting link below, and "8.2" works but "8.2.0" doesn't (just to make sure).

Also, I would be interested in knowing where it comes from, for the sake of learning.